modules/ROOT/pages/mulesoft-tcat-archetypes.adoc

```diff
@@ -4,4 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: tcat, archetypes
 
-See xref:mulesoft-tomcat-archetypes.adoc[MuleSoft Tomcat Archetypes].
```

Comment:
Can we remove this page?
I think this should be a page-alias in mulesoft-tomcat-archetypes.adoc